applications/FluidDynamicsApplication/custom_utilities/estimate_dt_utilities.h

```diff
@@ -221,8 +249,9 @@ class EstimateDtUtility
     ///@name Member Variables
     ///@{
 
-    double              mCFL; // User-defined CFL number
-    double              mDtMax; // User-defined maximum time increment allowed
+    double              mCFL;        // User-defined CFL number
+    double              mDtMax;      // User-defined maximum time increment allowed
+    double              mDtMin;      // User-defined minimum time increment allowed
     ModelPart::Pointer  mpModelPart; // The problem's model part
```

Comment:
Using shared_ptr (ModelPart::Pointer) is ok for python but gives a serious problem in cpp. The reason is the fact that submodelparts are stored by raw pointers and there is no way to get a Pointer from them. So I would suggest to use a normal pointer. Also it is arguable (and we can discuss it as another issue) if I should keep the sub modelparts by Pointer.